spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/servlet/JspTemplateAvailabilityProvider.java

```diff
@@ -38,11 +40,17 @@
 	public boolean isTemplateAvailable(String view, Environment environment,
 			ClassLoader classLoader, ResourceLoader resourceLoader) {
 		if (ClassUtils.isPresent("org.apache.jasper.compiler.JspConfig", classLoader)) {
-			String resourceName = getResourceName(view, environment);
-			if (resourceLoader.getResource(resourceName).exists()) {
-				return true;
+
+			try {
+				String resourceName = getResourceName(view, environment);
+				if (resourceLoader.getResource(resourceName).exists()) {
+					return true;
+				}
+				return new File("src/main/webapp", resourceName).exists();
```

Comment:
The scope of this `try` block is broader than necessary. It only needs to contain this line.

Reply:
I updated by your suggestion.

Reply:
I updated by your suggestion.